packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -202,9 +226,205 @@ export default async function initSanity(
   // Ensure we are using the output path provided by user
   outputPath = answers.outputPath
 
+  if (initNext) {
+    const useTypeScript = await prompt.single<boolean>({
+      type: 'confirm',
+      message: `Would you like to use TypeScript?`,
+      default: true,
+    })
+
+    const fileEnding = useTypeScript ? 'ts' : 'js'
+
+    const embeddedStudio = await prompt.single({
+      type: 'confirm',
+      message: `Would you like an embedded Sanity Studio?`,
```

Comment:
This question is a bit unclear, it doesn't say what an embedded Sanity Studio is but also doesn't explain what my choices are and what will happen if I say no. So it's really hard to make this decision if you don't know what will happen next. 

I think we should make it more clear what does an embedded Sanity Studio mean  and what is the decision I am making so user understand what will happen depending on which option they choose to continue with.

Reply:
You are right in that we should provide more context around some of the questions. I'll have a think on how we may be able to better do this while prompting questions.